src/services/asset-discovery-service.ts

```diff
@@ -173,8 +173,14 @@ export class AssetDiscoveryService extends PercyClientService {
   // We shouldn't bother passing on requests that will never be saved
   shouldProcessRequest(resourceUrl: string, rootResourceUrl: string): boolean {
     const parsedRootResourceUrl = new URL(rootResourceUrl)
+    const parsedResourceUrl = new URL(resourceUrl)
     const rootUrl = `${parsedRootResourceUrl.protocol}//${parsedRootResourceUrl.host}`
 
+    // Only capture resources with a proper protocol we support capturing
+    if (parsedResourceUrl.protocol === 'http:' || parsedResourceUrl.protocol === 'https:') {
```

Comment:
Do you think it's better to parse the URL for the protocol, or use a regexp or string method?

```js
(/^https?:/).test(resourceUrl)
// or
resourceUrl.startsWith('http:') || resourceUrl.startsWith('https:')